src/manifest.json

```diff
@@ -15,6 +15,9 @@
       "js": ["scripts/main.js"]
     }
   ],
+  "permissions": [
+    "tabs"
```

Comment:
I was getting an error where the background couldn't respond to main thread. Although I think I worked out what it was and it was not that. So I'll undo that.